### ISSUE TITLE
RangeError: WebAssembly.instantiate(): Out of memory

### ISSUE THREAD

**Issue by StevenXL** [cross-seed/cross-seed#782]
### Have you checked for any existing open/closed tickets that are the same or similar to your issue?

- [X] I've searched far and wide!

### I have gone to the Discord server for support and/or acknowledge that this is a bug and not a support issue

- [X] I've tried to solve this already

### What's happening surrounding the bug? (context)

Hi folks. I seem to be running into the same issue as #178 . Like in that issue, I am running cross-seed on a seedbox (UltraSeedBox to be precise). I don't have the ability to run `ulimit -v unlimited`. Below is the stack trace:

```
2024-08-29 14:58:45 info: cross-seed v6.0.0-32
2024-08-29 14:58:45 info: Validating your configuration...
node:internal/deps/undici/undici:5560
      return await WebAssembly.instantiate(mod, {
                               ^

RangeError: WebAssembly.instantiate(): Out of memory: Cannot allocate Wasm memory for new instance
    at lazyllhttp (node:internal/deps/undici/undici:5560:32)

Node.js v20.17.0
```

I noticed that in issue #178 , a downgrade seemed to have helped. This makes me wonder if this is a regression. 

### What should be happening?

n/a

### Your setup

```markdown
Node.js v20.17.0
UltraSeedBox
```

### config.js (REDACT SENSITIVE INFORMATION)

n/a

### verbose logs

2024-08-29 14:58:45 info: cross-seed v6.0.0-32
2024-08-29 14:58:45 info: Validating your configuration...
2024-08-29 14:58:45 verbose: [rtorrent] Calling method download_list with params []

### Failure or refusal to provide the information we require to address bugs will result in potential closure of the issue.

- [X] I acknowledge that i have searched existing issues, read the documentation and faq, read the logs to see if the answer was provided in error messaging, and attempted to get support via the Discord prior to submitting this issue.

**Comment by mmgoodnow** [comment]
The last comment on that issue has the new fix—Upgrade to node 22.2.0 or greater and run with  `NODE_OPTIONS=--disable-wasm-trap-handler` as an environment variable.

**Comment by zakkarry** [comment]
https://www.cross-seed.org/docs/basics/faq-troubleshooting#rangeerror-webassemblyinstantiate-out-of-memory-error-when-starting

Covered on our documentation.

**Comment by StevenXL** [comment]
> The last comment on that issue has the new fix—Upgrade to node 22.2.0 or greater and run with `NODE_OPTIONS=--disable-wasm-trap-handler` as an environment variable.

Sorry for not doing a better job searching. Thank you for the reply. Great work on this program!

**Comment by zakkarry** [comment]
> > The last comment on that issue has the new fix—Upgrade to node 22.2.0 or greater and run with `NODE_OPTIONS=--disable-wasm-trap-handler` as an environment variable.
> 
> Sorry for not doing a better job searching. Thank you for the reply. Great work on this program!

Thanks, we appreciate any and all love shown from the community and our users.

No big deal about the searching, while the docs are rather well-written (if i do say so myself :P) and complete at this point, they are still extremely dense :D
